README.rst

```diff
@@ -2902,10 +2976,15 @@ If you want to use a specfic connection object for the remote retrieval, you can
 
     $local_cluster = new Cluster("cluster/539xxxxxxxxxxxxxxxxxxxx18", $api);
 
-For set default storage if you have storage unset in api->
+For set default storage if you have storage unset in::
```

Comment:
do you really need to assing this to an attribute to $api?

Reply:
I think this should actually read "For set default storage if you have storage unset in api: $local_cluster = new Cluster("cluster/539xxxxxxxxxxxxxxxxxxxx18", null, storagedirectory);" I am a little unsure about the `storagedirectory` variable, and will investigate what actually works.